src/react-apps/package.json

```diff
@@ -15,8 +15,8 @@
     "jest-environment-enzyme": "^7.0.1",
     "jest-enzyme": "^7.0.1",
     "lerna": "^3.4.3",
-    "react": "^16.6.3",
-    "react-dom": "^16.6.3",
```

Comment:
Why downgrade? And remove the `^`?

Reply:
Upgraded to 16.6.3 on all apps and hard coded version.